FirstAssig.java

```diff
@@ -4,6 +4,6 @@ public static void main (String[] args)
   {
   
   //write a print statement with your name
-  
+  System.out.println("Rich");
```

Comment:
Good Work Here...Remember print would have done the same thing since you are only printing one line in this whole program